docs/distributions/nixos/installation.md

```diff
@@ -185,15 +210,41 @@ If you would like to organize your configuration a little better, check out othe
 
 ## Wi-Fi and Bluetooth setup
 
+### Pre-Installation Steps
+
+Choose a method below and follow the [Wi-Fi and Bluetooth Guide on macOS](../../guides/wifi-bluetooth.md#on-macos), then come back.
+
+=== "Method 1"
+    Method 1 requires you to run the firmware script twice, both in macOS and Linux. If you have uninstalled macOS, you may still be able to obtain the firmware files via a 600MB macOS recovery image without reinstalling macOS. See the linked guide for more information.
+
+=== "Method 2"
+    This method moves the firmware renaming step to macOS instead of Linux, where no scripts will need to be ran. This method is more robust but requires some manual configuration.
```

Comment:
This method moves the firmware renaming step to macOS instead of Linux, so no scripts will need to be run on the latter. This method is more robust, but it requires some manual configuration.